static_frame/core/index.py

```diff
@@ -1107,13 +1107,15 @@ def __contains__(self, value: tp.Any) -> bool:
     # utility functions
 
     def unique(self,
-            depth_level: DepthLevelSpecifier = 0
+            depth_level: DepthLevelSpecifier = 0,
+            order_by_occurrence: bool = False,
```

Comment:
Should the default be `True`?

Reply:
I guess I was aiming for backwards compatibility here, so I wanted to retain the earlier behavior. Would you like this done by default now?